DEVCVL-263 - Default network BSC mainnet when in PROD or UAT | Loading issue resolved

shakil-adv opened the pull request.

kyawthetkt commented:
![image](https://user-images.githubusercontent.com/25482318/169007520-af0813b0-2784-4987-a6db-31881406e50c.png)
If logged in, two drop down of chain appears.

kyawthetkt commented on src/components/header/Header.js:
```diff
@@ -18,6 +18,13 @@ const Header = () => {
+        window.location.href === "https://marketplace.canvasland.io/" ||
+        window.location.href === "http://marketplace1.canvasland.io/"
+    ) {
+        switchNetwork("0x38");
```
`window.location.href === "https://marketplace.canvasland.io/" ||
        window.location.href === "http://marketplace1.canvasland.io/"`

For `http://marketplace1.canvasland.io/`,it I think should be BSC Testnet.

jibrail-idris replied:
Simple and straightforward.

shakil-adv commented on src/components/header/Header.js:
```diff
@@ -26,8 +26,6 @@ const Header = () => {
-        console.log(key);
-        await enableWeb3();
         switchNetwork(key);
     };
```
I've removed that, somehow it is back after the manual merge. Removed again

shakil-adv commented:
> ![image](https://user-images.githubusercontent.com/25482318/169007520-af0813b0-2784-4987-a6db-31881406e50c.png) If logged in, two drop down of chain appears.

I've removed that, somehow it is back after the manual merge. Removed again
